src/app/components/reportesDiarios/resumenDiarioMensual.html

```diff
@@ -0,0 +1,195 @@
+<div class="row">
+    <div class="col">
+        <plex-box type="primary">
+
+            <table *ngIf="reporte.dias.length>0" class="table table-striped table-bordered">
```

Comment:
```suggestion
            <table *ngIf="reporte?.dias?.length" class="table table-striped table-bordered">
```
Esto me explotó porque reporte o dias venía nulo.
Esta sintaxis es una abreviatura de 
`reporte && reporte.dias && reportes.dias.length`

Si reporte tiene algo distinto de null, undefined y false pasa a analizar reporte.dias y así va avanzando
El `.length` devuelve verdadero ya si es mayor a 0 por eso se puede omitir el `> 0`

Reply:
Como reporte no tiene nunca días, el componente o explota o no se muestra con mi cambio.